scripts/triggers-end-to-end-tests/tests.ts

```diff
@@ -264,4 +264,80 @@ describe("import/export end to end", () => {
 
     expect(true).to.be.true;
   });
+
+  it("should be able to import/export rtdb data", async function() {
+    // eslint-disable-next-line no-invalid-this
+    this.timeout(2 * TEST_SETUP_TIMEOUT);
+    await new Promise((resolve) => setTimeout(resolve, 2000));
+
+    // Start up emulator suite
+    const emulatorsCLI = new CLIProcess("1", __dirname);
+    await emulatorsCLI.start(
+      "emulators:start",
+      FIREBASE_PROJECT,
+      ["--only", "database"],
+      (data: unknown) => {
+        if (typeof data != "string" && !Buffer.isBuffer(data)) {
+          throw new Error(`data is not a string or buffer (${typeof data})`);
+        }
+        return data.includes(ALL_EMULATORS_STARTED_LOG);
+      }
+    );
+
+    // Write some data to export
+    const config = readConfig();
+    const aRef = admin
+      .database()
+      .refFromURL(`http://localhost:${config.emulators!.database.port}/?ns=namespace-a`);
+    await aRef.set({
+      ns: "namespace-a",
+    });
+    const bRef = admin
+      .database()
+      .refFromURL(`http://localhost:${config.emulators!.database.port}/?ns=namespace-b`);
+    await bRef.set({
+      ns: "namespace-b",
+    });
```

Comment:
nit: Can we also try to read from (but not write to) `namespace-c` and assert that namespace-c is not exported?

Reply:
Sue!